truffle/src/main/java/org/jruby/truffle/nodes/arguments/CheckArityNode.java

```diff
@@ -47,8 +47,8 @@ public void executeVoid(VirtualFrame frame) {
         final Object[] frameArguments = frame.getArguments();
         final int given;
         final RubyBasicObject keywordArguments;
-        
-        //TODO (MS): Check merge 
+
+        //TODO (MS): Check merge
```

Comment:
ah sorry I misread the diff, I thought the diff added the extra spaces.
Removing neighboring style issues is fine.